src/Domain/CoordinateMaps/Wedge3D.cpp

```diff
@@ -56,11 +63,21 @@ Wedge3D::Wedge3D(const double radius_inner, const double radius_outer,
                           (1.0 - sphericity_inner) * radius_inner);
   sphere_rate_ = 0.5 * (sphericity_outer_ * radius_outer -
                         sphericity_inner * radius_inner);
+  if (UNLIKELY(with_logarithmic_map_)) {
```

Comment:
Is this unlikely?

Reply:
Yeah I think the UNLIKELY's are unnecessary in this PR, since the condition is either true or false for a given instance of the class. I don't think it makes a difference in the constructor, but for anything that will be called multiple times like the jacobian`, it will always be true/false for that instance so won't be UNLIKELY at all. (I'm opened to be corrected on my reasoning here)

Reply:
I'm not sure how UNLIKELY/ branch prediction works, but in my gsl rootfinder PR, I was similarly asked to put UNLIKELY in if statements that are only called once and that depends are things that are known at the initialization of the root finding, I'm open to being corrected as well

Reply:
All the UNLIKELY annotations in #771 are either failure cases (which had better be unlikely) or checks for high verbosity (which must be off for real runs so we're not flooded with output).  I don't see any reason to believe either case is generally unlikely here.